docs/user-manual/broker-plugins.adoc

```diff
@@ -178,3 +178,30 @@ In the example below `ROLE_PROPERTY` is set to `permissions` when that property
    </broker-plugin>
 </broker-plugins>
 ----
+
+== Using the ConnectionPeriodicExpiryPlugin
+
+The `ConnectionPeriodicExpiryPlugin` will implement a global expiry (and disconnect) for connections that live longer than `periodSeconds` on a matching acceptor basis.
+
+This plugin can be useful when credential rotation or credential validation must be enforced at regular intervals as authentication will be enforced on reconnect.
+
+The plugin requires the configuration of the `acceptorMatchRegex` to determine the acceptors to monitor. It is typical to separate client acceptors and federation or cluster acceptors such that only client connections will be subject to periodic expiry. The `acceptorMatchRegex` must be configured to match the name of the acceptor(s) whose connections will be subject to periodic expiry.
+
+|===
+| Property | Property Description | Default Value
+
+|`acceptorMatchRegex`|the regular expression used to match against the names of acceptors to monitor | ""
+|`periodSeconds`|the max duration or period, in seconds, that a connection can last | 15 minutes (as seconds)
```

Comment:
the default would be more obvious if offered in the way it is to actually be used, e.g:  "900 (i.e 15 minutes)"

Reply:
fair point, thanks!